src/pages/Thanks/ThanksPage.vue

```diff
@@ -176,9 +179,32 @@ export default {
 					checkoutId: transactionId,
 					visitorId: cookieStore.get('uiv') || null,
 				}
-			}).then(() => {
+			}).then(({ data }) => {
+				// Get teamId from receipt
+				const receipt = data?.shop?.receipt ?? null;
+
+				const loansResponse = receipt?.items?.values ?? [];
+				const loans = loansResponse
+					.filter(item => item.basketItemType === 'loan_reservation')
+					.map(item => {
+						return {
+							...item.loan,
+							team: item.team,
+						};
+					});
+
+				const teamsIds = loans.filter(loan => !!loan?.team?.id)
```

Comment:
Not critical to change, but the logic to get `loans` and `teamIds` is now duplicated in this component. Two small methods could be added to the component and shared.